### DISCUSSION TITLE
When to use SharePoint search and when to use Microsoft search

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2863] [Q&A]
In most of the cases i use SharePoint Search inside the Search Results web part. but recently i am reading that Microsoft Search is more future proof.. so is this correct? or those 2 options provides the same capabilities ?

**Comment by wobba** [comment]
If all you need is SharePoint content, then using the SharePoint API might be simpler. If you want personalized ranking of the results, then using the Microsoft Graph API is better.  So it depends on your scenario for using search API's.

**Reply by kasperbolarsen** [reply]
If you are working with people search I believe some properties like skills, prior projects and interests will only be available using SharePoint as a source. If you need to access the AAD extended properties then MS search is the right choice.
